libraries/AP_Scripting/examples/servo_override.lua

```diff
@@ -0,0 +1,21 @@
+-- Testing set_output_pwm_chan_timeout_ms
+--
+-- This will set MAIN1 servo to 1700 pwm for 1 second,
+-- then normal behavior for 1 second, and then 1100 pwm for 1 second.
+
+local flipflop = true
+
+function update()
+    if flipflop then
+        SRV_Channels:set_output_pwm_chan_timeout_ms(0, 1700, 1000)
```

Comment:
timeout is 1 second, but duration isn't spec'd here

Reply:
Timeout is duration in this case. I'm not following you otherwise.

Reply:
Oh, I was confused, and wasn't paying enough attention to your description of behavior. Since you don't update for 2 seconds, the timeout occurs after 1 second and control reverts from the script to normal. I was thinking the timeout was only meant as a safeguard in case the script stopped, but it functions as an override duration too.

Reply:
Right! It might be handy in a case where (this is a terrible design, please no one do this) when dropping something out the back of the aircraft you need to make sure that the elevator is neutral or above the hinge line to prevent it from hitting it. It also allows for cases of holding an output while other things happen and no extra instructions have to be used in scripting space.